src/main/java/seedu/address/logic/commands/list/ListApplicationCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands.list;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.util.CommandUtil.getCommandResult;
+import static seedu.address.model.FilterableItemList.PREDICATE_SHOW_ALL_ITEMS;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.ui.tabs.TabName;
+
+/**
+ * todo javadocs
+ */
+public class ListApplicationCommand extends ListCommand {
+
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_SUCCESS = "Listed all applications";
```

Comment:
As all our list commands are going to have similar success messages, perhaps we could do what we did with the rest of the commands which would be to add something like this after line 25: `String listSuccessMessage = String.format(MESSAGE_LISTED_ITEMS, APPLICATION_NAME);` where `MESSAGE_LISTED_ITEMS` resides in Messages.java and goes something like: `"Listed all %1$s items";`? Just a suggestion!

Reply:
oh this sounds great, will abstract it into messages.java then